sorts/double_sort.py

```diff
@@ -14,29 +17,32 @@ def double_sort(lst):
     >>> double_sort([-3, 10, 16, -42, 29]) == sorted([-3, 10, 16, -42, 29])
     True
     """
-    no_of_elements = len(lst)
+    no_of_elements = len(collection)
     for _ in range(
         int(((no_of_elements - 1) / 2) + 1)
     ):  # we don't need to traverse to end of list as
         for j in range(no_of_elements - 1):
             if (
-                lst[j + 1] < lst[j]
+                collection[j + 1] < collection[j]
             ):  # applying bubble sort algorithm from left to right (or forwards)
-                temp = lst[j + 1]
-                lst[j + 1] = lst[j]
-                lst[j] = temp
+                collection[j], collection[j + 1] = collection[j + 1], collection[j]
             if (
-                lst[no_of_elements - 1 - j] < lst[no_of_elements - 2 - j]
+                collection[no_of_elements - 1 - j] < collection[no_of_elements - 2 - j]
             ):  # applying bubble sort algorithm from right to left (or backwards)

```

Comment:
Let's not wrap lines of code just to make room for a trailing comment.  Please put the comment on a line of its own _before_ the `if` statement.  Please repeat on any other comments in this file that force a line to be wrapped.
```suggestion
            ):  # applying bubble sort algorithm from right to left (or backwards)
```